addons/jest/src/components/Panel.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/default-props-match-prop-types */
```

Comment:
some eslint plugins got updated and starting throwing warnings about propTypes, we should get rid of all prop-types IMHO